bases/bot_detector/api_private/src/api/v4/highscore.py

```diff
@@ -0,0 +1,61 @@
+import logging
+from collections import defaultdict
+from datetime import datetime
+
+from bot_detector.api_private.src.app.repositories import ScraperDataRepo
+from bot_detector.api_private.src.app.views.response import (
+    ActivityView,
+    ScraperDataView,
+    SkillView,
+)
+from bot_detector.api_private.src.core.fastapi.dependencies.session import (
+    get_session,
+)
+from bot_detector.database.repositories.hiscore import HighscoreDataLatestRepo
+from bot_detector.structs import HighscoreDataLatestStruct
+from fastapi import APIRouter, Depends, Query
+
+logger = logging.getLogger(__name__)
+
+router = APIRouter()
+
+
+def convert_latest_struct_to_scraper_data_view(
```

Comment:
for the v4 i'd not convert, and keep it the `HighscoreDataLatestStruct` we only convert the v3 model to keep backwards compatibility, the ML component uses the `/highscore/latest` route